hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -512,6 +519,15 @@ public HoodieTableVersion getTableVersion() {
         : VERSION.defaultValue();
   }
 
+  /**
+   * @return the hoodie.table.initial.version from hoodie.properties file.
+   */
+  public HoodieTableVersion getTableInitialVersion() {
+    return contains(INITIAL_VERSION)
+            ? HoodieTableVersion.versionFromCode(getInt(INITIAL_VERSION))
```

Comment:
Is the config value an integer or the String representing the `HoodieTableVersion` enum?

Reply:
INITIAL_VERSION is similar to VERSION in type.